scripts/check_fork_node_wiki.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import sys
+
+from modularodm import Q
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.project.model import Node
+from website.addons.wiki.model import NodeWikiPage
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def get_fork_projects():
+    return Node.find(
+        Q('is_fork', 'exists', True) &
+        Q('is_deleted', 'eq', False)
+    )
+
+
+def main():
+    counts = 0
+    for node in get_fork_projects():
+        if node.wiki_pages_versions:
+            for key in node.wiki_pages_versions:
+                for wiki_id in node.wiki_pages_versions[key]:
+                    wiki = NodeWikiPage.load(wiki_id)
```

Comment:
Extension of above:
```python
wiki = database.nodewikipage.find_one({'_id': wiki_id}, {'node': 1})
```

`wiki['node']` would be the current `wiki.node._id`. Everything else would be the same, just with dict-style accesses.